### PR TITLE
add contributing workflow

### PR DESCRIPTION
**Pull request by LouisSchmieder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,21 @@ Required dependencies:
 Gitly will support Postgres and MySQL in the future (once V ORM does).
 
 ![](https://user-images.githubusercontent.com/687996/85933714-b195fe80-b8da-11ea-9ddd-09cadc2103e4.png)
+
+
+## Contributing
+Workflow:
+1. Fork the project
+1. Add your feature
```
Comment (Delta456) [inline README.md:50]: Why everything is 1?

Comment (LouisSchmieder) [reply README.md:50]: It's an ordered list

Comment (Delta456) [reply README.md:50]: ok

**Comment by medvednikov** [comment]
If someone doesn't know how to use git/github, GitHub has a detailed article: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request

I don't see the point to repeat this information in our readme

**Comment by JalonSolov** [comment]
So this is a condensed version of CONTRIBUTING.md from vlang/v.  Any reason not to just copy that over?

**Pull request closed without merging**